iina/ViewLayer.swift

```diff
@@ -196,39 +196,6 @@ class ViewLayer: CAOpenGLLayer {
 
   // MARK: Utils
 
-  /** Check OpenGL error (for debug only). */
```

Comment:
As the comments indicates this is a debugging tool for developers and must not be removed. I'm thinking the correct change may be to use a Swift [Conditional Compilation Block](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/statements/#Conditional-Compilation-Block) to avoid including dead code. See what the senior developers think.

Reply:
@low-batt for the time being, I just returned the old code: c6e3bbc32ef70daf08139d84cfb6828ce61aebaa

Reply:
Sounds good.